File: web3-libs/mod2/playground/aula2/connect.js
// 1. Instale e importe a biblioteca Web3.js
import Web3 from "web3";

// 2. Configure o provider
const RPC_URL = "http://127.0.0.1:8545";
const web3 = new Web3(RPC_URL);

// 3. Verifique a conexão
web3.eth.getBlockNumber().then((number) => {
  console.log(`Block number ${number}`);
});

File: web3-libs/mod3/playground/aula2/connect.py
# 1. Instale e importe a biblioteca web3.py
from web3 import Web3

# 2. Configure o provider
RPC_URL = "http://127.0.0.1:8545"
web3 = Web3(Web3.HTTPProvider(RPC_URL))

# 3. Verifique a conexão
try:
    block_number = web3.eth.get_block_number()
    print(f"Número do bloco: {block_number}")
except Exception as error:
    print("Erro ao verificar a conexão:", error)
